FetchMigration/python/migration_monitor_params.py

```diff
@@ -4,4 +4,4 @@
 @dataclass
 class MigrationMonitorParams:
     target_count: int
-    dp_endpoint: str = "https://localhost:4900"
+    dp_endpoint: str
```

Comment:
consider data_prepper_endpoint.  Readers might assume that you meant "data plane".